R/pkg/R/functions.R

```diff
@@ -3531,6 +3583,26 @@ setMethod("explode",
             column(jc)
           })
 
+#' @details
+#' \code{sequence}: Generate a sequence of integers from start to stop, incrementing by step.
+#' If step is not set, incrementing by 1 if start is less than or equal to stop, otherwise -1.
+#'
+#' @param step an optional parameter to specify the step of the range.
+#' @rdname column_collection_functions
+#' @aliases sequence sequence,Column-method
+#' @note sequence since 2.4.0
+setMethod("sequence",
+         signature(),
```

Comment:
So, is this signature an issue? If that's difficult to address the comments, let's take out in this PR.

Reply:
Yes. I have problem with the above signature. I will remove ```sequence``` from this PR. Thanks.